### PR TITLE
AWS: add playbook templates

### PR DESCRIPTION
**Pull request by squioc**
Add two playbook templates using AWS triggers:

- One to forward AWS FlowLogs to SEKOIA.IO
- One to forward AWS CloudTrail to SEKOIA.IO

### CODE REVIEW

File: playbooks/templates/forward_aws_cloudtrail_records.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "name": "Push Events to Intake",
+      "type": "action",
+      "outputs": {
+        "default": []
+      },
+      "arguments": {
+        "events": "{{ node.0['records'] }}",
+        "intake_key": "eXErPtiqwZr71XioiAfTi4Fp6XzpaHv4"
```
Comment (LeoArurault) [inline playbooks/templates/forward_aws_cloudtrail_records.json:25]: Intake key is part of your personal configuration
```suggestion
```

File: playbooks/templates/forward_aws_cloudtrail_records.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Foward AWS cloudtrail records to SEKOIA.IO",
```
Comment (PierrickV) [inline playbooks/templates/forward_aws_cloudtrail_records.json:2]:
```suggestion
  "name": "Foward AWS CloudTrail records to SEKOIA.IO",
```

File: playbooks/templates/forward_aws_flowlogs_records.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Forward AWS Flowlogs records to SEKOIA.IO",
```
Comment (PierrickV) [inline playbooks/templates/forward_aws_flowlogs_records.json:2]:
```suggestion
  "name": "Forward AWS Flow Logs records to SEKOIA.IO",
```

File: playbooks/templates/forward_aws_flowlogs_records.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      "name": "Fetch Flowlog records",
```
Comment (PierrickV) [inline playbooks/templates/forward_aws_flowlogs_records.json:6]:
```suggestion
      "name": "Fetch Flow Logs records",
```

File: playbooks/templates/forward_aws_flowlogs_records.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      "arguments": {
+        "events": "{{ node.0['records'] }}",
+        "intake_key": ""
+      },
+      "action_uuid": "10314466-ac0c-4067-8e81-4093e3d1e1da",
+      "module_uuid": "92d8bb47-7c51-445d-81de-ae04edbb6f0a"
+    }
+  },
+  "workspace": "Operation Center",
+  "description": "This playbook pulls records from AWS Flowlogs then push the fetched records to SEKOIA.IO",
```
Comment (PierrickV) [inline playbooks/templates/forward_aws_flowlogs_records.json:32]:
```suggestion
  "description": "This playbook pulls records from AWS Flow Logs then push the fetched records to SEKOIA.IO",
```

File: playbooks/templates/forward_aws_cloudtrail_records.json (hunk 1 above)
Comment (squioc) [reply playbooks/templates/forward_aws_cloudtrail_records.json]: Good shot. I deleted the intake key from the testing platform 😁

**Review by PierrickV — APPROVED** [review]

**Pull request merged**